dolphinscheduler-ui/src/views/projects/task/components/node/types.ts

```diff
@@ -396,6 +398,8 @@ interface INodeData
   udfs?: string[]
   customConfig?: boolean
   mapping_columns?: object[]
+  useCustom?: boolean
+  runMode?: string
```

Comment:
No need to define 'useCustom' and 'runMode' here if you have already defined them in ITaskParams as the same types.

Reply:
done. thx.